masq/src/non_interactive_mode.rs

```diff
@@ -39,34 +39,34 @@ impl Main {
         for idx in 1..args_vec.len() {
             let one = &args_vec[idx - 1];
             let two = &args_vec[idx];
-            //tested by an integration test
-            if &args_vec[idx] == "--help" {
-                return Some(vec!["--help".to_string()]);
-            }
-            if !one.starts_with("--") && !two.starts_with("--") {
+            if !one.starts_with("--") && !two.starts_with("--")
```

Comment:
Make sure this works with the `--ui-port` parameter. Maybe refactor this function as well to make it clearer.